dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -442,14 +556,93 @@ object WavesDexCli extends ScoptImplicits {
           ),
         cmd(Command.CleanAssets.name)
           .action((_, s) => s.copy(command = Command.CleanAssets.some))
-          .text("Cleans levelDB cache with assets")
+          .text("Cleans LevelDb cache with assets")
           .children(
             opt[String]("dex-config")
               .abbr("dc")
               .text("DEX config path")
               .valueName("<raw-string>")
               .required()
               .action((x, s) => s.copy(configPath = x))
+          ),
+        cmd(Command.MockAsset.name)
+          .action((_, s) => s.copy(command = Command.MockAsset.some))
+          .text("Writes a mock value for this asset. This could be useful when there is asset from the stale fork")
+          .children(
+            opt[String]("dex-config")
+              .abbr("dc")
+              .text("DEX config path")
+              .valueName("<raw-string>")
+              .required()
+              .action((x, s) => s.copy(configPath = x)),
+            opt[String]("asset-id")
+              .abbr("aid")
+              .text("An asset id")
+              .valueName("<asset-id-in-base58>")
+              .required()
+              .action((x, s) => s.copy(assetId = x))
+          ),
+        cmd(Command.ListOrderBooks.name)
```

Comment:
Maybe, we should rename this command to ListAssetPair, because there is actually no orderbook printed, but asset pairs

Reply:
Fixed